### PR TITLE
chore: Add yarn global bin to the PATH in CI

### PR DESCRIPTION
**Pull request by ospencer**

### CODE REVIEW

**Review by phated — COMMENTED** [review]
@ospencer why doesn't this get added by setup-node? Seems like a problem that others would hit, right?

**Comment by ospencer** [comment]
Found out that on Windows, `yarn` uses `<prefix>\bin` but `npm` uses just `<prefix>`. I've updated this to just add `yarn global bin` to the PATH both on windows and unix systems instead of trying to match prefixes for npm and yarn.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -14,16 +14,26 @@ jobs:
 
     steps:
       - name: Setup Node.js
-        uses: actions/setup-node@v2.1.2
+        uses: actions/setup-node@v3.1.1
```
Comment (phated) [inline .github/workflows/ci.yml:17]:
```suggestion
        uses: actions/setup-node@v2.1.2
```
@ospencer let's keep this pinned until we want to do a full upgrade. I think our other workflows still have the v2

**Review by phated — APPROVED** [review]
LGTM! Thanks for going down this rabbit hole.

**Review by marcusroberts — APPROVED** [review]
Nice to keep the Windows platform equal!

**Pull request merged**